### DISCUSSION TITLE
Double click on format painter

### DISCUSSION THREAD

**Discussion by Kenroscommon** [valinet/ExplorerPatcher#2247] [Q&A]
Since loading EP, I cannot double click on the format painter in Microsoft products.  Each format click has to be individual.  Online suggestions for Excel say remove add-ins. But I have no add-ins working.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Kenroscommon** [reply]
Windows 11 21H2 OS build 22000.2416  EP 2261 1992 56 1  I changed no settings in Explorer Patcher.  Hope it helps, Thanks

**Reply by pyrates999** [reply]
In EP properties application, in **file explorer**, did you enable **register as a shell extension**?

You are also running windows 11 21h2.  Can you upgrade to windows 11 22h2?

You should also update EP.  You are running an older version.  Open the EP properties application, go to **updates**, and click on **update program and restart file explorer**.

**Comment by Kenroscommon** [comment]


**Reply by pyrates999** [reply]
Your mobile phone number is showing.  I would log into github instead of replying via email.

**Reply by pyrates999** [reply]
This comment here still is showing it.  Log into github and edit your comment here.

**Reply by pyrates999** [reply]
@Amrsatrio think you could delete this comment as it contains his phone number?

**Comment by Kenroscommon** [comment]
Thanks

***@***.*** -
________________________________

**Comment by Kenroscommon** [comment]
Lots of updates came through (EP & Windows) and the problem went away.  Thanks for your help.

**Reply by pyrates999** [reply]
Welcome.
